packages/_shared/src/FieldConnector.ts

```diff
@@ -95,6 +97,13 @@ export class FieldConnector<ValueType> extends React.Component<
     }
   );
 
+  componentDidUpdate(prevProps: FieldConnectorProps<ValueType>) {
+    if (prevProps.isDisabled !== this.props.isDisabled) {
+      // eslint-disable-next-line react/no-did-update-set-state
```

Comment:
Is it necessary to disable the lint rule here? Can't this be done with `getDerivedStateFromProps`?